impl/src/main/java/io/jsonwebtoken/impl/DefaultJwtParser.java

```diff
@@ -81,6 +81,32 @@
 
     private long allowedClockSkewMillis = 0;
 
+    /**
+     * @deprecated for backward compatibility only, see other constructors.
```

Comment:
Please add a TODO to remove for 1.0.  We don't guarantee semantic versioning in the `impl` module, but we can keep this around until 1.0 just in case.

Reply:
We should also have a TODO to make these fields `final` when we remove the default constructor.

Reply:
+100